src/superio/ite/it8659e/superio.c

```diff
@@ -22,6 +23,13 @@ static void it8659e_init(struct device *dev)
 		res = probe_resource(dev, PNP_IDX_IO0);
 		if (!conf || !res)
 			break;
+
+		/* Force HWM/PECI logic reset on warm boot */
```

Comment:
This doesn't actually check for warm boot, so maybe the comment should say something like
```suggestion
		/* Always force HWM/PECI logic reset to avoid it getting stuck on after a warm boot */
```

Also, is the reset bit defined in `IT8659_H_V0.7.2_industrial_20190917.pdf` or that's a wrong datasheet?